packages/z3solver/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: z3solver
+  version: 4.9.1.0
+source:
+  url: https://files.pythonhosted.org/packages/91/be/d247b751fd779e2e6abb175347cf97893e8606a19deaeab4ed26581b81a0/z3-solver-4.9.1.0.tar.gz
+  sha256: 5ed51c9748a23b6e21af9a526fa21f7ff0a76fd696aff436676300f162e10ddc
+build:
+  ldflags: -s SIDE_MODULE=1
```

Comment:
Your build has bigger problems, but to avoid the error about ``pkg_resources`` being unavailable in the import test, you have to add
```
requirements:
  run:
    - setuptools
  ```
and any other packages that are required to import the package.